### PR TITLE
Bug fix/6441 after running a business flow a new is added at the end

### PR DESCRIPTION
**Pull request by ravirk91**
Bug fix/6441 after running a business flow a new is added at the end

### CODE REVIEW

File: Ginger/Ginger/App.xaml.cs [hunk 1]
```diff
@@ -927,12 +927,14 @@ private static void LoadRecentBusinessFlow()
                 }
 
                 //load new Business Flow as default
+               // TODO: Need to check is actually below LoadDefaultBussinessFlow()method is required here
                 App.BusinessFlow = LoadDefaultBusinessFlow();
```
Comment (MeniKadosh1) [inline Ginger/Ginger/App.xaml.cs]: LoadRecentBusinessFlow() indeed needs to be changed now that we don't have Automate tab.
please change it as following: 
check if there are more than 0 BF's if yes so set App.BusinessFlow with first BF in list, if not having so use LoadDefaultBusinessFlow() to add defualt BF and set it to  App.BusinessFlow

File: Ginger/GingerCore/Repository/RepositorySerializer.cs [hunk 2]
```diff
@@ -489,7 +489,8 @@ private static object xmlReadObject(Object Parent, XmlReader xdr, RepositoryItem
 
                 if (obj == null)
                 {
-                    throw new Exception("Error:Cannot create Class: " + ClassName);
+                      Reporter.ToLog(eLogLevel.INFO, "Error:Cannot create Class: " + ClassName, writeOnlyInDebugMode: true);
+                      return null;
```
Comment (MeniKadosh1) [inline Ginger/GingerCore/Repository/RepositorySerializer.cs]: issue with this is that we might miss that there is some issue with some class, at least change the Log to be: eLogLevel.Error and remove this writeOnlyInDebugMode: true- it always needs to be added to Log

File: Ginger/Ginger/App.xaml.cs (hunk 1 above)
Comment (ravirk91) [reply Ginger/Ginger/App.xaml.cs]: it is already handled in code. 
image

Removing the TODO comment.

File: Ginger/GingerCore/Repository/RepositorySerializer.cs (hunk 2 above)
Comment (ravirk91) [reply Ginger/GingerCore/Repository/RepositorySerializer.cs]: changed the Log from info to error and removed writeOnlyInDebugMode: true.

**Comment by MeniKadosh1** [comment]
Yaron is fixing the serilizer and i already fixing the LoadRecentBusinessFlow on diffrent branch so closing this branch and related ticket.

**Comment by MeniKadosh1** [comment]
Ravi please delete the branch

**Pull request closed without merging**